src/localDate.js

```diff
@@ -12,7 +12,7 @@ function throwNotSupported() {
 function getDateTime(dateString) {
   const match = rISO8601.exec(dateString);
   if (match) {
-    const [, y, M, d, h, m, s, ms, zoneInfo] = match;
+    const [, y, M, d, h, m, s, __, ms, zoneInfo] = match;
```

Comment:
`__` Is there a dateString containing the matching string?

Reply:
__ is `.`.

Reply:
I understand. It's a meaningless variable!

Reply:
Yeah, it's removed.